src/routes/home/Home.js

```diff
@@ -505,14 +525,11 @@ class Home extends React.Component {
         this.alert(
           <React.Fragment>
             <p>
-              Could not extract plate and/or location and/or date from{' '}
-              {fileCopy} Please enter/confirm any missing values manually.
+              Could not extract the {missingValuesString} from {fileCopy} Please
+              enter/confirm any missing values manually.
             </p>
-
-            <p style={{ whiteSpace: 'pre-wrap' }}>{err.stack}</p>
           </React.Fragment>,
         );
-        console.error(err.stack);
```

Comment:
This will make it slightly harder to debug, since the stack trace won't tell me which line failed, so I just got rid of it. Need to think about how to debug the extraction of particular fields if/when needed: maybe just "revert" this change by changing `Promise.allSettled` back to `Promise.all` and not doing anything with the result?

Reply:
8b105528c7cea9d270490d091198e03222890588 should take care of this. It won't show the error stack to the user, but that one isn't very useful anyway due to code minification. However, the source-mapped stack will be in the console, which is useful for debugging.